### ISSUE TITLE
How to get rid of opening splash?

### ISSUE THREAD

**Issue by PlaguedHikikomori** [KuromeSan/chovy-gm#17]
Hello, 

tried to delete that annoying green splash at the beginning, and just noted Karoshi have it too.

Tried to find out if that string was somewhere in the C# code, but atm nothing found.

Could it be hardcoded in EBOOT.BIN?

Thanks

**Comment by nkrapivin** [comment]
Yes, it is hardcoded into the runner, as a DXT4 (or DXT5) texture.

**Comment by KuromeSan** [comment]
> Yes, it is hardcoded into the runner, as a DXT4 (or DXT5) texture.

At what offset? I could add an option to change it ..

**Comment by PlaguedHikikomori** [comment]
> 
> 
> Yes, it is hardcoded into the runner, as a DXT4 (or DXT5) texture.

![immagine](https://user-images.githubusercontent.com/34514202/81397443-a10da700-9127-11ea-8ddd-46b8869dd4e9.png)

Had this thought too, but either the runner and the compiled .iso will show true gibberish when dropped in a DXT3/5 texture viewer. When the other texture of the game are clearly visible.

Also tried to make a couple hex compares, and setting some breakpoints in ppsspp when the texture splashes, still nothing easily editable found.

**Comment by KuromeSan** [comment]
You probably would need to decrypt the EBOOT.BIN file into EBOOT.ELF then encrypt it again with sign-np.

 PPSSPP has an option to do the decryption iirc

**Comment by PlaguedHikikomori** [comment]
[EBOOT.zip](https://github.com/KuromeSan/chovy-gm/files/4599084/EBOOT.zip)

Here's the picture-free runner, if some other people need that. Well done guys.

**Comment by KuromeSan** [comment]
nice! @PlaguedHikikomori id like to add the option to change ti to chovy-gm at some point tbh
